docs/content/concepts/layer/control/catalog/sp800-53rev5-example.md

```diff
@@ -0,0 +1,340 @@
+            <li>segregation of access control roles, e.g. access request, access authorization, access administration;</li>
+            <li>requirements for formal authorization of access requests;</li>
+            <li>requirements for periodic review of access rights;</li>
+            <li>removal of access rights;</li>
+            <li>archiving of records of all significant events concerning the use and management of user identities and secret authentication information;</li>
+            <li>roles with privileged access.</li>
+        </ol>
+    </p>
+    <p><strong>Other information</strong></p>
+    <p>Care should be taken when specifying access control rules to consider:
+        <ol type="a">
+            <li>establishing rules based on the premise "Everything is generally forbidden unless expressly permitted" rather than the weaker rule "Everything is generally permitted unless expressly forbidden";</li>
+            <li>changes in information labels that are initiated automatically by information processing facilities and those initiated at the discretion of a user;</li>
+            <li>changes in user permissions that are initiated automatically by the information system and those initiated by an administrator;</li>
+            <li>rules which require specific approval before enactment and those which do not.</li>
+        </ol>
+    </p>
+    <p>Access control rules should be supported by formal procedures and defined responsibilities.</p>
+    <p>Role based access control is an approach used successfully by many organizations to link access rights with business roles.</p>
+    <p>Two of the frequent principles directing the access control policy are:
+        <ol type="a">
+            <li>Need-to-know: you are only granted access to the information you need to perform your tasks (different tasks/roles mean different need-to-know and hence different access profile);</li>
+            <li>Need-to-use: you are only granted access to the information processing facilities (IT equipment, applications, procedures, rooms) you need to perform your task/job/role.</li>
+        </ol>
+    </p>
+</blockquote>
+
+**NOTE:** Some of the reference links have been removed in the above examples as compared to the original source. This was done to simplify the text for illustration.
+
+## Comparing the ISO 27002 and NIST SP 800-53 Examples
+
+Comparing the ISO 27002 and NIST SP 800-53 examples illustrates obvious differences. NIST SP 800-53 includes several components, such as references and control enhancements that ISO 27002 does not. NIST SP 800-53's statement of the control itself is also much more detailed and specific than ISO 27002 because it provides those details in its implementation guidance instead of the control text. There are other differences in terminology as well, such as NIST SP 800-53 using the term "supplemental guidance" for roughly what ISO 27002 calls "other information".
```

Comment:
This is minor, and feel free to disregard, but using the word "components" here given `component`s in OSCAL document instances? "Syntax features" is a rough attempt but I am sure others can recommend a better alternative.

```suggestion
Comparing the ISO 27002 and NIST SP 800-53 examples illustrates obvious differences. NIST SP 800-53 includes several syntax features, such as references and control enhancements that ISO 27002 does not. NIST SP 800-53's statement of the control itself is also much more detailed and specific than ISO 27002 because it provides those details in its implementation guidance instead of the control text. There are other differences in terminology as well, such as NIST SP 800-53 using the term "supplemental guidance" for roughly what ISO 27002 calls "other information".
```

Reply:
I would go for "structures" but they aren't actually syntax features, they are entities within the OSCAL model, any of which might be expressible using different syntax.

Reply:
Changed the language in a subsequent commit: https://github.com/wendellpiez/OSCAL/commit/5658aee5cad89ae6e38331e77b24323254684549